tests/serializers/models/data.py

```diff
@@ -1,18 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
Are you in the habit of adding this even if there aren't any non-ASCII chars? I guess it's obsolete once we drop Python 2, since the default there is utf-8, right?